src/Core/Framework/Plugin/Command/PluginCreateCommand.php

```diff
@@ -66,12 +67,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         try {
             $pluginName = $input->getArgument('plugin-name');
+            $staticPrefix = $input->getOption('static') ? 'static-' : '';
 
             if (!$pluginName) {
                 $pluginName = $this->askPascalCaseString('Please enter a plugin name (PascalCase)', $io);
             }
 
-            $directory = $this->projectDir . '/custom/plugins/' . $pluginName;
+            $directory = $this->projectDir . "/custom/{$staticPrefix}plugins/" . $pluginName;
```

Comment:
```suggestion
            $directory = "{$this->projectDir}/custom/{$staticPrefix}plugins/{$pluginName}";
```
or use `sprintf`m same for the theme create command :-) Sorry, missed that before :-)

Reply:
Corrected with sprintf. Thanks